LightBulb/ViewModels/RootViewModel.cs

```diff
@@ -217,6 +223,12 @@ private void RegisterHotKeys()
             }
         }
 
+        private void RegisterDisplayState()
+        {
+            _powerBroadcastService.UnregisterAllBroadcasts();
+            _powerBroadcastService.RegisterDisplayOn(() => UpdateGamma(true));
```

Comment:
Calling `UpdateGamma` from here is risky as it can have race conditions with the timer. It's better to set some flag, e.g. `_isGammaStale = true` and then check this flag in `UpdateGamma()`.